src/diagnostics.rs

```diff
@@ -257,13 +283,14 @@
                     let path = info.location.uri.to_file_path().unwrap();
                     let filename = path.to_str().unwrap();
                     let p = get_kakoune_position_with_fallback(
+                        server,
                         filename,
                         info.location.range.start,
                         ctx,
                     );
                     format!(
-                        "{}:{}:{}: {}",
-                        short_file_path(filename, &ctx.root_path),
+                        "{}:{}:{}: ({server_name}) {}",
```

Comment:
I think we should only add the server name if there are multiple servers configured